libs/api/domains/vehicles/src/models/usersVehicles.model.ts

```diff
@@ -196,3 +199,60 @@ export class VehiclesList {
   })
   createdTimestamp?: string
 }
+
+@ObjectType()
+export class VehicleListed {
+  @Field({ nullable: true })
+  permno?: string
+
+  @Field({ nullable: true })
+  regno?: string
+
+  @Field({ nullable: true })
+  persidno?: string
+
+  @Field({ nullable: true })
+  role?: string
+
+  @Field({ nullable: true })
+  make?: string
+
+  @Field({ nullable: true })
+  colorCode?: string
+
+  @Field({ nullable: true })
+  colorName?: string
+
+  @Field({ nullable: true })
+  modelYear?: string
+
+  @Field(() => Boolean, { nullable: true })
+  requiresMileageRegistration?: boolean | null
+
+  @Field(() => Boolean, { nullable: true })
+  canRegisterMilage?: boolean | null
```

Comment:
Why can this be null? Isn't either true/false or unknown (undefined)?

Reply:
yeah makes sense 👍